### PR TITLE
Fix the TCOD export plugins for `PwCalculation` and `PhCalculation`

### LINKED ISSUE #365: Fix the TCOD exporters
The `BaseTcodTranslator` has been moved from `aiida-core` to `aiida-tcod`

### PR DESCRIPTION
**Pull request by sphuber**
Fixes #365 

The base translator class was moved from `aiida-core` to `aiida-tcod`.
Add the latter as dependency, fix the imports and drop the directory
from the pre-commit exclude list.

### CODE REVIEW

File: aiida_quantumespresso/tools/dbexporters/tcod_plugins/pw.py [hunk 1]
```diff
@@ -45,26 +49,26 @@ def get_software_package(cls,calc,**kwargs):
...
         """
         Returns the energy of defined type in eV.
         """
         parameters = calc.outputs.output_parameters
         if energy_type not in parameters.attrs():
             return None
         if energy_type + '_units' not in parameters.attrs():
-            raise ValueError('energy units for {} are '
-                             'unknown'.format(energy_type))
+            raise ValueError('energy units for {} are ' 'unknown'.format(energy_type))
```
Comment (borellim) [inline aiida_quantumespresso/tools/dbexporters/tcod_plugins/pw.py]: Maybe merge in a single string

**Comment by borellim** [comment]
Maybe a stupid question, but what happens if one does not install aiida-tcod? Does the error instruct to install the [tcod] extras?

File: aiida_quantumespresso/tools/dbexporters/tcod_plugins/pw.py (hunk 1 above)
Comment (sphuber) [reply aiida_quantumespresso/tools/dbexporters/tcod_plugins/pw.py]: Done

**Comment by sphuber** [comment]
> Maybe a stupid question, but what happens if one does not install aiida-tcod? Does the error instruct to install the [tcod] extras?
> 
> (PR ok otherwise for me)

Not a stupid question: I added a try except with a more informative error message.

**Comment by borellim** [comment]
Cool, thanks!

**Pull request merged**